R/print.R

```diff
@@ -53,19 +53,20 @@ print.gsplot <- function(x, ...){
 
   for (side.name in side.names){
     side <- as.side(side.name)
-    par(x[[side.name]]$par)
+    old.par <- par(x[[side.name]]$par)
     set_frame(views, side)
     if(!side %in% defined.sides){ 
       if(x[[side.name]][['axes']]){
-        Axis(side=side,x=lim(views, side))
+        do.call('Axis', append(list(side=side,x=lim(views, side)), config('axis')))
```

Comment:
what does having a do.call here do?? Isn't it just applied to one list element anyways?